sqlalchemy_redshift/commands.py

```diff
@@ -234,7 +239,15 @@ def __init__(self, select, unload_location, access_key_id=None,
                  manifest=False, delimiter=None, fixed_width=None,
                  encrypted=False, gzip=False, add_quotes=False, null=None,
                  escape=False, allow_overwrite=False, parallel=True,
-                 header=False, region=None, max_file_size=None):
+                 header=False, region=None, max_file_size=None,
+                 format_as_csv=False):
```

Comment:
It seemed simpler to add `format_as_csv` instead of adding a more generic `format` since UNLOAD only supports `CSV` at the moment.

Reply:
I'd prefer to keep our interfaces as consistent as possible. We already have a Format enum; I think we can just use that directly and let Redshift sort out whether the passed format is acceptable or not. It would give us forwards-compatibility if AWS implements additional formats from that existing enum in the future.

Reply:
Fair point.  How does 9b61436 look?  I tried to follow what the copy command was doing, but I think redshift supports CSV in copy statements a bit different than unloads (e.g., there doesn't appear to be a quote character option).